### PR TITLE
Option to limit episode length for play

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: alf/trainers/policy_trainer.py [hunk 1]
```diff
@@ -331,12 +334,13 @@ def play(root_dir,
...
         episode_reward += float(time_step.reward)
 
-        if time_step.is_last():
+        if time_step.is_last() or episode_length >= max_episode_length > 0:
             logging.info("episode_length=%s episode_reward=%s" %
                          (episode_length, episode_reward))
             episode_reward = 0.
             episode_length = 0.
             episodes += 1
+            time_step = env.reset()
```
Comment (hnyu) [inline alf/trainers/policy_trainer.py:343]: seems no need to return time_step

File: alf/trainers/policy_trainer.py [hunk 2]
```diff
@@ -267,6 +267,7 @@ def play(root_dir,
          checkpoint_step="latest",
          epsilon_greedy=0.1,
          num_episodes=10,
+         max_episode_length=0,
          sleep_time_per_step=0.01,
          record_file=None):
     """Play using the latest checkpoint under `train_dir`.
```
Comment (hnyu) [inline alf/trainers/policy_trainer.py:273]: docstring format needs to be updated

File: alf/trainers/policy_trainer.py (hunk 1 above)
Comment (emailweixu) [reply alf/trainers/policy_trainer.py:343]: in case it ends because of max_episode_length, the previous time_step is not correct.

Comment (hnyu) [reply alf/trainers/policy_trainer.py:343]: I mean reset() should be called but time_step will always be overwritten.

**Review by hnyu — APPROVED** [review]

File: alf/trainers/policy_trainer.py (hunk 1 above)
Comment (emailweixu) [reply alf/trainers/policy_trainer.py:343]: I don't understand what you mean.

Comment (hnyu) [reply alf/trainers/policy_trainer.py:343]: Sorry I misunderstood the code

**Pull request merged**